src/reducers/filter.js

```diff
@@ -0,0 +1,12 @@
+import { CHANGE_FILTER } from '../actions/index';
+
+const filterReducer = (state = '', action) => {
+  switch (action.type) {
+    case CHANGE_FILTER:
+      return action.payload.category;
+    default:
+      return state;
+  }
+};
```

Comment:
- [ ] Please ensure that the filter state initial value defaults to 'All' [screenshot from Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone4/ItDefaultsToAll.png)